main.go

```diff
@@ -406,7 +407,11 @@ func cmdEdit(c *cli.Context) error {
 			return err
 		}
 		files = filterMarkdown(files)
-		cmd := exec.Command(cfg.SelectCmd)
+		cmds, err := shellwords.Parse(cfg.SelectCmd)
+		if len(cmds) == 0 || err != nil {
```

Comment:
Thanks. Sorry, I should explain how to do this.

```go
	swp := shellwords.NewParser()
	swp.ParseEnv = true
	cmds, err := swp.Parse(cfg.SelectCmd)
	if len(cmds) == 0 || err != nil || swp.Position > 0 {
		return errors.New("selectcmd: invalid commands")
	}
```
go-shellwords can handle `|` or `<`, `>` againt shibukawa/shell can't handle it. However, in this case, those letters should not be handled. So please change to this.